app/helpers/course_details_helper.rb

```diff
@@ -71,4 +75,22 @@ def all_subjects
     @all_subjects ||= (SubjectSpecialism.pluck(:name) + AllocationSubject.pluck(:name) + [Trainees::Filter::ALL_SCIENCES_FILTER])
       .map(&:downcase).uniq.sort
   end
+
+  def all_start_years
+    @all_start_years ||= formatted_start_years
+  end
+
+  def formatted_start_years
+    AcademicCycle.order(start_date: :desc).map(&:start_year).each_with_object([]) do |start_year, array|
+      if current_academic_cycle_year?(start_year)
+        array << "#{start_year} - #{start_year + 1} (current year)"
+      else
+        array << "#{start_year} - #{start_year + 1}"
```

Comment:
can we use "to" instead of a hyphen please?

Reply:
Great! that should all be handled now. Let me know if there is anything else that needs changing.